pkg/resource/google/google_storage_bucket_test.go

```diff
@@ -0,0 +1,31 @@
+package google_test
+
+import (
+	"testing"
+
+	"github.com/cloudskiff/driftctl/test"
+	"github.com/cloudskiff/driftctl/test/acceptance"
+)
+
+func TestAcc_Google_StorageBucket(t *testing.T) {
+	acceptance.Run(t, acceptance.AccTestCase{
+		TerraformVersion: "0.15.0",
+		Paths:            []string{"./testdata/acc/google_storage_bucket"},
+		Args: []string{
+			"scan",
+			"--to", "gcp+tf",
+			"--filter", "Type=='google_storage_bucket'",
```

Comment:
Shouldn't we pin provider version to `3.78.0` in ACC tests ?

Reply:
We do not pin version in other ACC test so I followed the same behavior. 

The day we want to update the default provider version we can run the full ACC test suite on the new version if we do not pin version, and thus ensure backward compatibility since the version is pinned in the terraform files attached to theses tests. I think it can make sense to not pin them by default, an only pin them when we want to add an acceptance test  for a specific bug on a specific provider version. We may discuss this point with the team on discord after the summer break.

Reply:
We did it for some acceptance tests but not all of them, not sure if we should do it for all of them then